python/cudf/cudf/benchmarks/bench_cudf_io.py

```diff
@@ -82,3 +86,74 @@ def bench_json(benchmark, file_path, use_buffer, dtype):
         orient="records",
         dtype=dtype,
     )
+
+
+@pytest.mark.parametrize(
+    "dtype", ["float32", "float64", "int32", "int64", "str", "datetime64[s]"]
+)
+def bench_to_csv(benchmark, dtype):
+    cudf_df, file_path = create_cudf_dataset(
+        dtype, file_type="csv", only_file=False
+    )
+    benchmark(cudf_df.to_csv, file_path)
+
+
+@pytest.mark.parametrize(
+    "dtype", ["float32", "float64", "int32", "int64", "str", "datetime64[s]"]
+)
+def bench_from_csv(benchmark, use_buffer, dtype):
+    file_path = create_cudf_dataset(dtype, file_type="csv", only_file=True)
+    if use_buffer == "True":
+        with open(file_path, "rb") as f:
+            file = io.BytesIO(f.read())
+    else:
+        file = file_path
+    benchmark(cudf.read_csv, file)
+    os.remove(file_path)
+
+
+@pytest.mark.parametrize(
+    "dtype", ["float32", "float64", "int32", "int64", "str", "datetime64[s]"]
+)
+def bench_to_orc(benchmark, dtype):
+    cudf_df, file_path = create_cudf_dataset(
+        dtype, file_type="orc", only_file=False
+    )
+    benchmark(cudf_df.to_orc, file_path)
+
+
+@pytest.mark.parametrize(
+    "dtype", ["float32", "float64", "int32", "int64", "str", "datetime64[s]"]
+)
+def bench_read_orc(benchmark, use_buffer, dtype):
+    file_path = create_cudf_dataset(dtype, file_type="orc", only_file=True)
+    if use_buffer == "True":
+        with open(file_path, "rb") as f:
+            file = io.BytesIO(f.read())
+    else:
+        file = file_path
+    benchmark(cudf.read_orc, file)
```

Comment:
missing `os.remove(file_path)`

Reply:
Pandas does not have a `to_orc` function. I will be adding a condition to check if we will be running the pandas tests as well. If we are running the pandas test then this file will not be deleted here but in the pandas function for `read_orc` test